pkg/pipelines/bootstrap.go

```diff
@@ -60,6 +61,11 @@ func Bootstrap(quayUsername, baseRepo, prefix string) error {
 	}
 	outputs = append(outputs, dockerSecret)
 
+	tasks := tasks.GenerateTasks()
```

Comment:
This should probably be `tasks.Generate()` see the note about "stuttering" in https://blog.golang.org/package-names

Reply:
I've updated the function names.